typescript/infra/scripts/utils.ts

```diff
@@ -27,7 +28,7 @@ export function assertEnvironment(env: string): DeployEnvironment {
 
 export function getCoreEnvironmentConfig<Env extends DeployEnvironment>(
   env: Env,
-) {
+): CoreEnvironmentConfig<any> {
```

Comment:
Should this be `CoreEnvironmentConfig<Env>`?

Reply:
it should be `CoreEnvironmentConfig<EnvironmentChain<Env>>` but the type union doesn't resolve well